helm/templates/statefulset.yaml

```diff
@@ -64,12 +72,13 @@ spec:
               secretKeyRef:
                 name: {{ include "immudb.fullname" . }}-credentials
                 key: immudb-admin-password
-          {{- end}}      
+          {{- end}}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
           volumeMounts:
           - mountPath: /var/lib/immudb
             name: immudb-storage
+            subPath: immudb
```

Comment:
I've introduced a subPath since otherwise immudb would see a `lost+found` directory and assume it is a database

Reply:
This is a breaking change - we have to think about how to announce it then if someone is already using the helm chart. Maybe some automatic migration procedure? (although I'm not really in favor of such approach).

Reply:
I can add a compatibility flag maybe? So that the `subPath` clause is only added if something like `.values.volumeSubPath` is set to true. 
We set it to true in `values.yaml` and explain that if you already have some data either you migrate it (and we can also provide a guide to do that) or you set this value to false.

Reply:
I was thinking about other solutions here too:
1) Ignore lost+found (and maybe some other folders) in immudb itself
2) Add some init container to check if there's a subvolume but the main folder contains immudb.identifier file that would indicate that previously immudb files were stored in the main folder
3) Don't setup volumeSubPath by default but have a clear documentation why it should be set, maybe also allow specifying that in the values.yaml explicitly to any subfolder, not only `immudb` value?

Reply:
See commit e3005ef7 for that

Reply:
Solution 2 is complicated, I'd go with 1 or 3. 
I suggest to adopt the third.
We can have a `value.yaml` file like this:
```yaml
subPath:
  enabled: true # or false
  path: /immudb
```

and then use those values in the template. Plus add some documentations on when to use that (tipically, ext4 formatted block-based volumes, like DO or EBS)

Reply:
Commit 3eddd21a should address this.